Makefile

```diff
@@ -26,8 +27,18 @@ api-local:
 	@echo "\n******"
 
 web-local:
-	@echo "++\n***** Running Web in local Node server\n ++"
+	@echo "++\n***** Running Web in local Node server\n++"
 	@yarn workspace @ehpr/web start
-	@echo "\n*****"
+	@echo "++\n*****"
+
+start-local-services:
+	@echo "++\n***** Starting local services\n++"
+	@docker-compose up db
```

Comment:
Why just db? And can we add --build to this command

Reply:
here we are running underlying services specified in docker-compose not building anything.